### PR TITLE
T1021-develop: Adding mandatory on the invoice

### PR DESCRIPTION
**Pull request by lrasamoelina**
I added a mandatory statement on an invoice because since January 2013-the words: compensation for recovery costs,
 must be presented on an invoice.(regenerate the facture in format pdf)
htdocs/compta/facture.php

### CODE REVIEW

**Comment by rdoursenaud** [comment]
I think you're forgetting that Dolibarr is an international project not just for France.
The idea is good but the implementation should be more generic and flexible to allow taking care of local regulations.

Furthermore, the french regulation asks for « How much these will cost » not just stating that there will be some. To each business to decide it's own, even if we know accounting consultants generally give the same guidelines.

TL;DR: I vote against integration as-is.

**Comment by lrasamoelina** [comment]
yes,you have right but we can change this for just the facture of company which are in France

**Comment by rdoursenaud** [comment]
@lrasamoelina Yes, that would be a first step in the right direction. This should be enclosed on a big TODO though…

**Comment by lrasamoelina** [comment]
It's should be ok

**Comment by eldy** [comment]
The country must not be added before state.
Adding country is already available into dol_format_address function
